src/theme/index.tsx

```diff
@@ -338,4 +341,8 @@ html {
 a {
  color: ${({ theme }) => theme.deprecated_blue1}; 
 }
+
+:root {
+  ${({ theme }) => (theme.darkMode ? cssStringFromTheme(darkTheme) : cssStringFromTheme(lightTheme))}
```

Comment:
Will this conflict with existing css? Should it be restricted to the nft components?

Does sprinkles/vanilla offer an alternative to the custom cssStringFromTheme? This seems like unnecessary custom code.

Reply:
This does not override any existing styles so we have not found any conflicts. You only interact with the VE styles when imported directly.

 It does, but I wouldn't want to do that refactor until after we've had our long term architecture discussion which is out of scope for this PR https://vanilla-extract.style/documentation/styling-api/#style Currently the custom fn does allow for a very simple way to import in the themed styles and swap between light and dark theme